python/lsst/sconsUtils/dependencies.py

```diff
@@ -349,16 +422,27 @@ def CustomCppFlagCheck(context, flag, append=True):
     return result
 
 
-##
-# @brief A configuration test that checks whether the given source code
-#        compiles.
-# @param context    Configuration context.
-# @param message    Message disaplyed on console prior to running the test.
-# @param source     Source code to compile.
-# param  extension  Identifies the language of the source code. Use ".c" for C, and ".cc"
-#                   for C++ (the default).
-##
 def CustomCompileCheck(context, message, source, extension=".cc"):
+    """A configuration test that checks whether the given source code
+    compiles.
+
+    Parameters
+    ----------
+    context :
+        Configuration context.
+    message : `str`
+        Message displayed on console prior to running the test.
+    source : `str`
+        Source code to compile.
+    extension : `str`, optional
+        Identifies the language of the source code. Use ".c" for C, and ".cc"
+        for C++ (the default).
+
+    Returns
+    -------
+    result : `bool`
+        Did the flag work?
```

Comment:
"flag"?  Looks like this was copied from the function above.  Same goes for the next function.